charts/node-analyzer/templates/_helpers.tpl

```diff
@@ -113,6 +113,61 @@ Return the proper image name for the CSPM Analyzer
     {{- include "nodeAnalyzer.imageRegistry" . -}} / {{- .Values.nodeAnalyzer.kspmAnalyzer.image.repository -}} {{- if .Values.nodeAnalyzer.kspmAnalyzer.image.digest -}} @ {{- .Values.nodeAnalyzer.kspmAnalyzer.image.digest -}} {{- else -}} : {{- .Values.nodeAnalyzer.kspmAnalyzer.image.tag -}} {{- end -}}
 {{- end -}}
 
+{{/*
+ Helper to define skip of SSL Certificate for Runtime Scanner and Eve Connector
+*/}}
+{{- define "runtimeScanner.sslVerifyCertificate" -}}
+{{- if or (eq .Values.nodeAnalyzer.runtimeScanner.sslVerifyCertificate false) (eq .Values.nodeAnalyzer.sslVerifyCertificate false) (eq .Values.global.sslVerifyCertificate false) -}}
```

Comment:
I'm not sure about this, considering this situation
```yaml
global:
    sslVerifyCertificate: true

nodeAnalyzer:
    sslVerifyCertificate: false

    runtimeScanner:
        sslVerifyCertificate: true
```
for the runtime scanner we expect the SSL Cert verification to be enabled but in this context became false because there is at least one that is set to false... (same for all the other helpers)

Reply:
As discussed, changed the logic to:
```
{{/*
 Helper to define skip of SSL Certificate for Runtime Scanner and Eve Connector
*/}}
{{- define "runtimeScanner.sslVerifyCertificate" -}}
{{- $sslVerifyCertificate := coalesce (.Values.nodeAnalyzer.runtimeScanner.sslVerifyCertificate | quote) (.Values.nodeAnalyzer.sslVerifyCertificate | quote) (.Values.global.sslVerifyCertificate | quote) ("true" | quote) -}}
{{ $sslVerifyCertificate }}
{{- end -}}
```